src/string.cr

```diff
@@ -908,6 +927,12 @@ class String
     match[group]? if match
   end
 
+  # Returns *str* if *str* is found in this string.
+  #
+  # ```
+  # "crystal"["cry"]  # => "cry"
+  # "crystal"["ruby"] # => raises Nil assertion error
```

Comment:
```suggestion
  # "crystal"["ruby"] # raises Nil assertion error
```

Reply:
I don't think we usually wrap raised types in backticks.
```suggestion
  # "crystal"["ruby"] # raises NilAssertionError
```